### PR TITLE
[WFLY-12787] Document adding custom HTTP headers to the HTTP manageme…

### PR DESCRIPTION
**Pull request by darranl**
WildFly Core PR https://github.com/wildfly/wildfly-core/pull/4013

https://issues.jboss.org/browse/WFLY-12787

### CODE REVIEW

File: docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc [hunk 1]
```diff
@@ -61,6 +61,56 @@ Default URL
...
 
+=== Custom HTTP Headers
+
+For the responses returned from the HTTP management interface it is also possible to define custom constant HTTP headers that will be added to any response based on matching a configured prefix against the request path.
+
+As an example it could be desirable to add a HTTP header `X-Help` which points users to the correct location to obtain assistance.  The following management operation can be executed within the CLI to activate returning this header on all requests.
+
+[source,options="nowrap"]
+----
+/] /core-service=management/management-interface=http-interface: \
```
Comment (jmesnil) [inline docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc]: `\]` at the beginning of the command looks weird.

**Review by jmesnil — CHANGES_REQUESTED** [review]
Other than the command typo, LGTM

File: docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc (hunk 1 above)
Comment (darranl) [reply docs/src/main/asciidoc/_admin-guide/Management_Clients.adoc]: I can remove that if we want, was just showing the CLI command prompt before the command

**Comment by jmesnil** [comment]
This PR must be merged once the WildFly Core upgrade that incorporates https://github.com/wildfly/wildfly-core/pull/4013 is merged

**Comment by darranl** [comment]
@jmesnil Just added one more commit re the command prompt - is this good for you now?

**Review by jmesnil — APPROVED** [review]

**Comment by jmesnil** [comment]
@darranl thanks

**Pull request merged**